.gitignore

```diff
@@ -149,6 +149,9 @@ dmypy.json
 # pytype static type analyzer
 .pytype/
 
+# Python linter
+.ruff_cache/
```

Comment:
Our release process uses Cloud Build via `gcloud builds submit`. By default, this commands excludes files & directories from the build context from `.gitignore`.

This prevents:

```
ERROR: gcloud crashed (PermissionError): [Errno 13] Permission denied: '.ruff_cache/0.8.4/9619314862902349248'
```